client/src/app/site/pages/meetings/pages/agenda/modules/topics/modules/topic-poll/components/topic-poll-vote/topic-poll-vote.component.ts

```diff
@@ -382,4 +437,26 @@ export class TopicPollVoteComponent extends BasePollVoteComponent<ViewTopic> imp
             }
         }
     }
+
+    protected compareMinAllVotes(delegations: ViewUser[]): boolean {
+        let reachedMinValue = false;
+        const minVote = this.minVotes;
+        if (!(this.getVotingError() === `You do not have the permission to vote.`)) {
+            if (!this.hasAlreadyVoted()) {
+                if (this.getVotesCount() < minVote) {
+                    reachedMinValue = true;
+                }
+            }
+        }
+        for (let delegation of delegations) {
+            if (!(this.getVotingError(delegation) === `You do not have the permission to vote.`)) {
+                if (!this.hasAlreadyVoted(delegation)) {
+                    if (this.getVotesCount(delegation) < minVote) {
+                        reachedMinValue = true;
+                    }
+                }
+            }
+        }
```

Comment:
```suggestion
        for (let delegation of delegations.concat(this.user) {
            if (!(this.getVotingError(delegation) === `You do not have the permission to vote.`)) {
                if (!this.hasAlreadyVoted(delegation)) {
                    if (this.getVotesCount(delegation) < minVote) {
                        reachedMinValue = true;
                    }
                }
            }
        }
```

Skip the if clause by concatenating the operator user to the delegations and looping through that. You're doing the same thing in both clauses anyway.

You should probably also do this in the other places where you loop through the delegation after already having done the same thing for the operator user beforehand